generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -628,13 +641,13 @@ public class <%= entityClass %>ResourceIT <% if (databaseType === 'cassandra') {
 
     <%_ if (databaseType === 'sql' && isUsingMapsId === true) { _%>
     @Test
-    @Transactional
+    <%= transactionalAnnotation %>
```

Comment:
I think it should be on the same line here, as `transactionalAnnotation` contains the linebreak already.

Reply:
Yes, you are right